08-software-versions.Rmd

```diff
@@ -5,6 +5,108 @@
 ottrpal::set_knitr_image_path()
 ```
 
-```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
+## Learning Objectives
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "In this software versions chapter our learning objectives are to be able to: Recognize that software versions influence data analysis results and reproducibility. Record packages used for an analysis. Use renv to make an R environment shareable to collaborators"}
```

Comment:
could add another learning objective about recognizing other methods

Reply:
Adding `Recognize containerization as a method to share your entire computing environment with others`